solidity/test/test/TestSendReceiver.t.sol

```diff
@@ -111,4 +111,23 @@ contract TestSendReceiverTest is Test {
             "0x1234"
         );
     }
+
+    function testHandle_withHardcodedBody() public {
+        vm.expectRevert("failMessageBody");
+        testSendReceiver.handle(
+            0,
+            address(testSendReceiver).addressToBytes32(),
+            "failMessageBody"
+        );
+    }
+
+    function testHandle_withHardcodedBodyHex() public {
+        vm.expectRevert("failMessageBody");
+        testSendReceiver.handle(
+            0,
+            address(testSendReceiver).addressToBytes32(),
+            // failMessageBody but hex value
+            hex"6661696c4d657373616765426f6479"
```

Comment:
could you do
```
bytes memory hardcodedFail = "failMessageBody";
```
and use `hardcodedFail` as the param instead of the hex? just trying to avoid magic numbers

Reply:
sure